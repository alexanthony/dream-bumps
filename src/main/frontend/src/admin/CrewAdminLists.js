import React from 'react';
import {Grid, Cell} from 'material-grid/dist';

import CrewAdminList from './CrewAdminList';

export default () => (
  <Grid>
    <Cell col={6}>
      <h3>Men</h3>
      <CrewAdminList sex="male" />
    </Cell>
    <Cell col={6}>
      <h3>Women</h3>
      <CrewAdminList sex="female" />
    </Cell>
  </Grid>
)
