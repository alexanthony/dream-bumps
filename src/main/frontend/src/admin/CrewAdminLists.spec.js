import React from 'react';
import { shallow } from 'enzyme';
import CrewAdminLists from './CrewAdminLists';
import toJson from 'enzyme-to-json';

test('Renders properly', () => {
  const component = shallow(
    <CrewAdminLists />
  );
  expect(component).toBeDefined();
  expect(toJson(component)).toMatchSnapshot()
});
