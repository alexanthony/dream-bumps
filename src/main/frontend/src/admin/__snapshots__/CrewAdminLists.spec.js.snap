exports[`test Renders properly 1`] = `
<Grid>
  <Cell
    col={6}>
    <h3>
      Men
    </h3>
    <Connect(CrewAdminList)
      sex="male" />
  </Cell>
  <Cell
    col={6}>
    <h3>
      Women
    </h3>
    <Connect(CrewAdminList)
      sex="female" />
  </Cell>
</Grid>
`;
